### PR TITLE
Fixed incorrectly supported LocalOnly parm in OpenEnumerateInstances

### PR DESCRIPTION
**Pull request by andy-maier**
For details, see the commit message.
Ready for review.

### CODE REVIEW

File: pywbem/cim_operations.py [hunk 1]
```diff
@@ -4326,14 +4326,21 @@ def IterEnumerateInstances(self, ClassName, namespace=None,
             instances, as follows:
 
             * If `False`, inherited properties are not excluded.
-            * If `True`, inherited properties are basically excluded, but the
+            * If `True`, the behavior depends: If pull operations are used,
```
Comment (KSchopmeyer) [inline pywbem/cim_operations.py]: change "depends" to "is inconsistent"

Comment (KSchopmeyer) [inline pywbem/cim_operations.py]: Suggest change "depends" to "depends on underlying operation"

File: pywbem/cim_operations.py [hunk 2]
```diff
@@ -4326,14 +4326,21 @@ def IterEnumerateInstances(self, ClassName, namespace=None,
...
+              inherited properties are not excluded. If traditional operations
+              are used, inherited properties are basically excluded, but the
               behavior may be WBEM server specific.
-            * If `None`, this parameter is not passed to the WBEM server, and
+            * If `None`, the behavior depends: If pull operations are used,
```
Comment (KSchopmeyer) [inline pywbem/cim_operations.py]: Suggest change "depends" to "depends on underlying operation"

File: pywbem/cim_operations.py (hunk 1 above)
Comment (andy-maier) [reply pywbem/cim_operations.py]: Done

File: pywbem/cim_operations.py (hunk 2 above)
Comment (andy-maier) [reply pywbem/cim_operations.py]: Done

**Comment by andy-maier** [comment]
As discussed, updated the function test cases for Open...() to revert the change of removing the incorrect parameters, in order to verify that the parameters are actually ignored when specified.

**Comment by KSchopmeyer** [comment]
Whoops in approved

**Comment by andy-maier** [comment]
PR #1787 rolls this back into 0.14.5.

**Pull request merged**